src/TodoList.js

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import TodoListItem from "./TodoListItem";
+
+function TodoList(props) {
+
+
+  return (
+    <ul>
+
+      {props.todoList.map(function (item) {
```

Comment:
This could be simplified with an arrow function.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

`(item) => <TodoListItem
          item={item}
          key={item.id}
          onRemoveTodo={props.onRemoveTodo} />`